src/app/(sidebar)/transaction/sign/components/Overview.tsx

```diff
@@ -0,0 +1,286 @@
+                return (
+                  <div className={className} key={field.label}>
+                    <XdrPicker
+                      readOnly
+                      id={field.label}
+                      label={field.label}
+                      value={field.value.toString()}
+                    />
+                  </div>
+                );
+              } else {
+                return (
+                  <div className={className} key={field.label}>
+                    <TextPicker
+                      readOnly
+                      id={field.label}
+                      label={field.label}
+                      value={field.value.toString()}
+                      copyButton={{
+                        position: "right",
+                      }}
+                    />
+                  </div>
+                );
+              }
+            })}
+          </div>
+        </Card>
+      </div>
+
+      <div className="SignTx__Signs">
+        <div className="PageHeader">
+          <WithInfoText href="https://developers.stellar.org/docs/learn/encyclopedia/signatures-multisig">
+            <Text size="md" as="h1" weight="medium">
+              Signatures
+            </Text>
+          </WithInfoText>
+        </div>
+
+        <Card>
+          <div className="SignTx__Field">
+            <div className="full-width">
+              <MultiPicker
+                id="signer"
+                label="Add Signer"
+                value={secretInputs}
+                onUpdate={onUpdateSecretInputs}
+                validate={validate.secretKey}
+                placeholder="Secret key (starting with S) or hash preimage (in hex)"
+              />
+            </div>
+
+            <div className="full-width">
+              <TextPicker
+                id="bip-path"
+                label="BIP Path"
+                placeholder="BIP path in format: 44'/148'/0'"
+                onChange={(e) => {
+                  setBipPath(e.target.value);
+
+                  const error = validate.bipPath(e.target.value);
+
+                  if (error) {
+                    setBipPathErrorMsg(error);
+                  }
+                }}
+                error={bipPathErrorMsg}
+                value={bipPath}
+                note="Note: Trezor devices require upper time bounds to be set (non-zero), otherwise the signature will not be verified"
+              />
+            </div>
+
+            <div className="SignTx__Buttons">
+              <div>
+                <Button
+                  disabled={!HAS_SECRET_KEYS}
+                  size="md"
+                  variant="secondary"
+                  onClick={() =>
+                    signTxWithSecretKeys(
+                      sign.importXdr,
+                      secretInputs,
+                      network.passphrase,
+                    )
+                  }
+                >
+                  Sign with secret key
+                </Button>
```

Comment:
We should enable the sign button only when signatures are valid.

![image](https://github.com/stellar/laboratory/assets/7346473/703acb4e-320b-4c8c-b865-74e4a0fe880b)